data/governments.txt

```diff
@@ -47,6 +47,21 @@ government "Bounty Hunter"
 	"fine" 0
 	"hostile hail" "hostile bounty hunter"
 
+# Bounty hunters that the player's escorts will attack and vice-versa
+government " Bounty Hunter "
```

Comment:
Not using `"display name"`?

Reply:
They need specific attitudes for the missions to work, and no government presently has that attitude. If the government name, `" Bounty Hunter "` may cause problems, I can change it to something else and use `display name`.

Reply:
I'll test that later today.

Reply:
What I think Anarchist meant is, no matter the name you give it, you should always give it the intended display name in order to display normally in game. See: https://github.com/endless-sky/endless-sky/issues/4767

Reply:
The string, `" Bounty Hunter "` is too generic for the purpose the government serves. It is specifically there to hunt the player and the Hai, but nobody else. I'm going to give them a longer name like `Bounty Hunters that Fight Hai` and then use `display name` to make it show up as `Bounty Hunters`.